doc/classes/SoftBody3D.xml

```diff
@@ -100,11 +101,13 @@
 			[b]Note:[/b] Object A can detect a contact with object B only if object B is in any of the layers that object A scans. See [url=$DOCS_URL/tutorials/physics/physics_introduction.html#collision-layers-and-masks]Collision layers and masks[/url] in the documentation for more information.
 		</member>
 		<member name="damping_coefficient" type="float" setter="set_damping_coefficient" getter="get_damping_coefficient" default="0.01">
+			The body's damping coefficient. Higher values will slow down the body more noticeably when forces are applied.
 		</member>
 		<member name="disable_mode" type="int" setter="set_disable_mode" getter="get_disable_mode" enum="SoftBody3D.DisableMode" default="0">
 			Defines the behavior in physics when [member Node.process_mode] is set to [constant Node.PROCESS_MODE_DISABLED]. See [enum DisableMode] for more details about the different modes.
 		</member>
 		<member name="drag_coefficient" type="float" setter="set_drag_coefficient" getter="get_drag_coefficient" default="0.0">
+			The body's drag coefficient. Higher values make the body bouncier.
```

Comment:
Is this correct? Drag is normally about air resistance

In fact it seems it's not actually used anywhere at least not in the standard physics

Reply:
It... seems to be used for SoftBody3D, but could this be placebo? As physics are, the code is not exactly clear.

No, you're right, this was a slip on my part. This has got to be about air resistance.

Reply:
Nowhere in the code is this used as of right now, seems to have been used in Bullet but not implemented yet here

Reply:
Oh boy.